### ISSUE TITLE
WooCommerce - Products Widget widths override

### ISSUE THREAD

**Issue by colbyalbo** [elementor/elementor#16048]
The column widths for the products widget is being overidden by WC styles

**Comment by datverse** [comment]
Same here, It have from version 3.4.0. We can see when change to Tablet mode, Mobile mode: The widths for products is 48%
I use a css trick to hotfix:

```
@media only screen and (max-width: 768px) {
    .woocommerce ul.products[class*=columns-] li.product, .woocommerce-page ul.products[class*=columns-] li.product {
        width: auto;
    }
}
```

**Comment by d19dotca** [comment]
I believe this is a duplicate of https://github.com/elementor/elementor/issues/16001

**Comment by shilo-ey** [comment]
I merged this issue with #16001, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!

**Comment by martinhaslinger1** [comment]
Hello everbody,

I just ran into the same error again using Elementor Pro Version 3.7.7 in combination with the Storefront theme. Any ideas why the bug appeared again and a way to solve it?

Thanks in advance!

![image](https://user-images.githubusercontent.com/115837940/195933916-e23be1ec-38b2-47a7-8f79-3fa2d07bc3fc.png)
